es-metadata.yml

```diff
@@ -1,8 +1,8 @@
 schemaVersion: 0.0.1
-isProduction: true
```

Comment:
This is testifled by @jeffschwMSFT that this repo is production.

Reply:
sure

Reply:
Thanks!